client/zenith/client/config.py

```diff
@@ -79,6 +79,10 @@ class Config:
     forward_to_host: str = "localhost"
     #: The port to forward tunnel traffic to
     forward_to_port: int = 8000
+    #: The period after which a server alive message will be sent if no data is received
+    server_alive_period: conint(gt = 0) = 10
+    #: The number of server alive messages that can fail before the tunnel is terminated
+    server_alive_failures: conint(gt = 0) = 3
```

Comment:
Is 3 failures too many to wait? does that mean waiting 40 seconds to timeout?

Reply:
This means 30s. I think this is reasonable TBH but happy to reduce.

Reply:
Note that if the connection is properly terminated by either end, this doesn't apply - the process exits immediately. This is just to cover the case where the server end is silently dropped with no notification to the client.

Reply:
Yeah, I was thinking it means wiat 10 seconds, then fail, and fail on the fourth. I was thinking only allowing one failure seems OK, but this is all better than forever!